edx-platform/pearson-vue-theme/lms/templates/index.html

```diff
@@ -0,0 +1,61 @@
+<%page expression_filter="h"/>
+<%inherit file="main.html" />
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.urls import reverse
+
+from openedx.core.djangolib.markup import HTML, Text
+%>
+
+<div class="background-login"></div>
+
+<div class="login-container">
+  <div>
+    <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
+  </div>
+
+  <div class="signin-container">
+
+    <div class="toggle-form">
+      <h1>${_("Welcome!")}</h1>
+      <br>
+      <span class="text">${_("Sign in or create an account to access your courseware.")}</span>
+
```

Comment:
@luismorenolopera is this blank line necessary? same with line 12, 17 and 19.

Reply:
@ivanvgh I think we can leave the space of line 12, the rest I have removed